include/boost/geometry/strategies/geographic/buffer_point_circle.hpp

```diff
@@ -55,77 +59,67 @@ template
 class geographic_point_circle
 {
 public :
+
     //! \brief Constructs the strategy
-    //! \param count number of points for the created circle (if count
-    //! is smaller than 3, count is internally set to 3)
-    explicit geographic_point_circle(std::size_t count = 90)
-        : m_count((count < 3u) ? 3u : count)
+    //! \param points_per_circle Number of points for a full circle
+    //! (if points_per_circle is smaller than 4, it is internally set to 4)
+    explicit geographic_point_circle(std::size_t points_per_circle = 90)
+        : m_points_per_circle((points_per_circle < 4u) ? 4u : points_per_circle)
```

Comment:
what is the point of changing this from 3 to 4? Is this for robustness?

Reply:
One had 3, the other had 4. I liked to harmonize it. I can make both `3` too. Either choice is fine, it's very low anyway, not really circle-like.

Reply:
For the join it was also 3 but that was erroneous, I made it minimum 4 to get well-shaped symmetric joins.
I don't think it was ever used with 3 (default = 90)